DQM/Integration/python/clients/beamhlt_dqm_sourceclient-live_cfg.py

```diff
@@ -48,6 +48,30 @@
 if unitTest:
   process.load("DQM.Integration.config.unittestinputsource_cfi")
   from DQM.Integration.config.unittestinputsource_cfi import options
+
+  # Overwrite source of the unitTest to use a streamer file instead of the DAS query output
+  print("[beamhlt_dqm_sourceclient-live_cfg]:: Overriding DAS input to use a streamer file")
+
+  # Read streamer files from https://github.com/cms-data/DQM-Integration
+  import os
+  dqm_integration_data = [os.path.join(dir,'DQM/Integration') for dir in os.getenv('CMSSW_SEARCH_PATH','').split(":") if os.path.exists(os.path.join(dir,'DQM/Integration'))][0]
```

Comment:
@francescobrivio , please use `DQM/Integration/data`

Reply:
`data` was originally added by me beacuse I manually added a `data` directory inside cmssw/DQM/Integration, the new `cms-data` repo does not have the `data` subdir.
Will the fileInPath automatically resolve it to the correct cms-data repo?

Reply:
yes the `cms-data/SubSystem-Package` contents are installed as `somepath/SubSystem/Package/data` (where `<somepath>` is added to `CMSSW_SEARCH_PATH` env . This is done to match the `cmssw/src/SubSystem/Package/data` .

Reply:
For `cms-data/DQM-Integration`, the install path has
```
> ls /cvmfs/cms-ci.cern.ch/week0/PR_b061bf72/slc7_amd64_gcc10/cms/data-DQM-Integration/V00-00-00/DQM/Integration/data 
README.md  run346373
```

Reply:
@smuzaffar thank you very much for the explanation! 
I'll fix the path in the next commit.

Reply:
addessed in [d310c28](https://github.com/cms-sw/cmssw/pull/37667/commits/d310c2820b6de1fb7017e5c34064a0834b6673c4)